src/Model/EventModel.php

```diff
@@ -4,6 +4,7 @@
 
 use Klaviyo\Exception\KlaviyoException;
 use Klaviyo\Model\ProfileModel;
+use DateTime;
```

Comment:
nit: let's do something similar to our app code organization and import standard classes above the package classes

Reply:
can separate with a newline too if you want